benchmark_runner/jupyterlab/templates/analyze_workloads/analyze_vdbench_logs/analyze_vdbench_logs.py

```diff
@@ -0,0 +1,219 @@
+import csv
+import os
+import pandas as pd
+
+# display df
+from IPython.display import display
+
+# display bokeh
+from bokeh.plotting import figure, show
+from bokeh.io import output_notebook, output_file
+from bokeh.models import HoverTool
+
+from typeguard import typechecked
+
+from benchmark_runner.jupyterlab.templates.logs_operations.logs_operations import LogsOperations
+
+
+class AnalyzeVdbenchLogs:
+    """
+    This class analyzes Vdbench logs
+    """
+    def __init__(self, s3_logs_url: str):
+        self.__workload = 'vdbench'
+        self.s3_logs_url = s3_logs_url
+        self.__logs_operations = LogsOperations(s3_logs_url= self.s3_logs_url )
+
+    @staticmethod
+    @typechecked
+    def is_csv_file(file_path: str):
+        """
+        This method checks if file is csv
+        @param file_path:
+        @return:
+        """
+        try:
+            with open(file_path, 'r') as file:
+                dialect = csv.Sniffer().sniff(file.read(1024))
+                return dialect.delimiter == ',' or dialect.delimiter == ';'
+        except csv.Error:
+            return False
+
+    def get_vdbench_csv_log_files(self):
+        """
+        This method returns vdbench csv log files path
+        @return: vdbench csv log files path
+        """
+        workload_dir_name = self.__logs_operations.get_workload_dir(workload=self.__workload)
+        # Iterate over all files in the directory
+        workload_files_path = []
+        for file_name in os.listdir(workload_dir_name):
+            file_path = os.path.join(workload_dir_name, file_name)
+            if os.path.isfile(file_path) and self.is_csv_file(file_path) and self.__workload in file_name:
+                workload_files_path.append(file_path)
+        return workload_files_path
+
+    def display_df(self):
+        """
+        This method display dataframe
+        @return:
+        """
+        csv_files = self.get_vdbench_csv_log_files()
+        for csv_file in csv_files:
+            # Read the CSV file
+            df = pd.read_csv(csv_file)
+            df = df[df["Run"] != "fillup"]
+            print(f"display file: {csv_file.split('/')[-1]}")
+            display(df)
+
+    def display_latency_horizontal(self):
+        """
+        This method displays vdbench latency horizontal
+        @return:
+        """
+        output_notebook()
+
+        for csv_file in self.get_vdbench_csv_log_files():
+            # Read the CSV file
+            df = pd.read_csv(csv_file)
+            df = df[df["Run"] != "fillup"]
```

Comment:
Can you make these strings as const? I saw so many repetitions. So it is better to make const.
RUN = "Run"
FILLUP =  "fillup"
RESP = "Resp"

Reply:
I prefer to keep it it with original names instead of using constants because its the original name in dataframe